src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -207,6 +208,19 @@ public boolean processACHBatchDetails() {
         
         return failedRowsErrors;
     }
+    
+    protected void cleanPayeeACHAccountExtractDetail(PayeeACHAccountExtractDetail detail) {
+        if (StringUtils.isNotBlank(detail.getBankAccountNumber()) && !StringUtils.isNumeric(detail.getBankAccountNumber())) {
+            String logMessageStarter = "cleanPayeeACHAccountExtractDetail, the bank account for " + detail.getNetID();
+            if (StringUtils.contains(detail.getBankAccountNumber(), KFSConstants.DASH)) {
+                LOG.error(logMessageStarter + " contains dashes, so removing them");
+                String cleanedBankAccount = StringUtils.remove(detail.getBankAccountNumber(), KFSConstants.DASH);
+                detail.setBankAccountNumber(cleanedBankAccount);
+            } else {
+                LOG.error(logMessageStarter + " is not numberic but does NOT contains dashes, no automatic cleaning available");

```

Comment:
Minor typo in log message; should change from "...numberic..." to "...numeric..."